src/App.tsx

```diff
@@ -1,28 +1,69 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { FilterStatusType, Todo } from './types/Todo';
+import * as tadoService from './api/todos';
+import { Footer } from './component/Footer';
+import { Error } from './component/Error';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { TodoItem } from './component/Todo';
+
+function filterTodos(todoArr: Todo[], filter: FilterStatusType) {
+  const todos = [...todoArr];
+
+  switch (filter) {
+    case FilterStatusType.All:
+      return todos;
+    case FilterStatusType.Active:
+      return todos.filter(todo => todo.completed === false);
+    case FilterStatusType.Completed:
+      return todos.filter(todo => todo.completed === true);
+    default:
+      return todos;
+  }
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterStatusType>(
+    FilterStatusType.All,
+  );
+
+  const [hiddenError, setHiddenError] = useState(true);
+
+  useEffect(() => {
+    tadoService
+      .getTodos()
+      .then(setTodos)
+      .catch(error => {
+        setHiddenError(false);
+        alert(error);
+      })
+      .finally(() => {
+        setTimeout(() => {
+          setHiddenError(true);
+        }, 3000);
```

Comment:
If you don't have an error so you don't need to reset it. You can create a handler that sets an error and then clean it in 3 seconds, it will help you to don't repeat it in the future. Something like this:
```typescript
const handleError = (errorMessage: string) => {
  setErrorMessage(errorMessage);

  setTimeout(() => {
    setErrorMessage('');
  }, 3000);
};
```